test/e2e/.gitignore

```diff
@@ -0,0 +1,2 @@
+kind-install-for-capd.sh
```

Comment:
Why is this needed?

Reply:
The file is downloaded from cluster-api github repo so I don't want to include it in our repo.